### DISCUSSION TITLE
Reinforcement Learning Based Fault Tolerant Control of a Quadrotor (Project 235)

### DISCUSSION THREAD

**Discussion by robertogl** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#71] [Collaborate]
Contribute to the discussion by asking and/or answering questions, commenting, or sharing your ideas for solutions to [project #235](https://github.com/mathworks/MathWorks-Excellence-in-Innovation/tree/main/projects/Reinforcement%20Learning%20Based%20Fault%20Tolerant%20Control%20of%20a%20Quadrotor)

---
*:robot: An AI bot may reply to your comments with suggestions and guidance. It will not solve the project for you. Your comments may be processed by a third-party LLM. To direct a question to a specific person, tag them with @username.*

**Comment by d1skidder** [comment]
Hi, I am a senior at Lynbrook Highschool in San Jose and interested in the application of reinforcement learning for autonomous vehicles and drones. I look forward to working and collaborating on this project

**Comment by a-b-i-y** [comment]
Hello,
I am Abiy. I have been working on this project. And i have progressed a lot
(i think). But lately i have been busy with work and stalled it.

I am very pleased to work with you. I will share my progress with you.

Thank you very much.

Regardes,

Abiy Amare Bereda

Assistant Researcher 1 at Ethiopian Biotechnology Institutes ( ebti )

Emerging Technology Center

Reverse Engineering Directorate

E-mail : ***@***.***

contact : +251921895278

Addis Ababa, Ethiopia
